datafusion/sql/src/select.rs

```diff
@@ -311,6 +311,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 Ok(LogicalPlan::Filter(Filter::try_new(
                     filter_expr,
                     Arc::new(plan),
+                    20,
```

Comment:
Probably reading from SQL context is the right thing to do here? i.e. a variable?